code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Unitals.hs

```diff
@@ -61,7 +61,7 @@ outputSymbols = map qw [QP.position, QP.velocity, QM.orientation,
 unitalChunks :: [UnitalChunk]
 unitalChunks = [QP.acceleration, QP.angularAccel, QP.gravitationalAccel, 
   QP.impulseV, QP.impulseS, iVect, jVect, normalVect, QP.distance, QP.displacement, 
-  QP.time, QP.angularDisplacement, posCM, posj, massj, mTot, accj, velj,
+  QP.time, QP.angularDisplacement, posCM, posj, massj, mTot, accj, angAccj, velj,
```

Comment:
For example, how did you find out that `angAccj` was 'missing' from the list of unital chunks that should be "registered" for GamePhysics?

Reply:
So that's the "real" issue. We need to look at `angAccj`, `IM:rotMot` and the model create off of `angAccj` to see what's going on.